src/main/webapp/i18n/de/participation.json

```diff
@@ -41,7 +41,7 @@
             "student": "Student",
             "team": "Team",
             "students": "Studierende",
-            "practiceMode": "Übungsmodus",
+            "practice": "Praxis",
```

Comment:
I don't think this translation is a fitting change from "Übungsmodus" for "practice" 🤔  Maybe "Üben"?

Reply:
I prefer "üben" to Praxis

Reply:
I agree that Praxis does not fit. The German translation is a bit difficult here. We should probably keep "Übungsmodus" here